SPECS/guest-templates-json.spec

```diff
@@ -12,6 +12,7 @@ Source1000: almalinux-8.json
 Source1001: almalinux-9.json
 Source1002: centos-stream-8.json
 Source1003: oel-9.json
+Patch1000: 0001-Remove-preview-from-4-json-files.patch
```

Comment:
We usually separate our patches from upstream
```suggestion
Source1003: oel-9.json

# XCP-ng patches
Patch1000: 0001-Remove-preview-from-4-json-files.patch
```

Reply:
Theey are not with upstream but with Sources we added. But yes I can add another line to separate more.

Reply:
no you're right
didn't see that in GH diff - leave as is